website/customers.html

```diff
@@ -1,6 +1,6 @@
 ---
-title: Redash Customers
-description: Great collaction of our users
+title: Redash customers
+description: Great collection of our customers
```

Comment:
Let's change it to: "Empowering data-driven decision making in thousands of teams across the globe. From small teams to huge organizations, Redash helps tens of thousands of users to query and visualize data in a new and meaningful way."

Reply:
I stopped using title case by the way so I'd stick to Redash customers